deliverables/README.md

```diff
@@ -9,60 +9,123 @@ To answer questions, please just put the answer on the next line after the quest
 #### HTML
-What are classes and IDs (and how are they different)?  
+What are classes and IDs (and how are they different)?
+They both indicate differt parts of your webpage that can be modified using CSS or Javascript. When one manipulates a class using CSS or Javascript they are modifying all the Ids within that class. When one edits a ID using CSS or Javascript they only manipulate one specific element of a webpage. 
+
 What are elements?  
+HTML elements are individual components of a webpage. They tend to mark the begining and end of a webpage. 
+
 What are tags?  
+HTML tags apply meaning to elements of a webpage and help apply context to the actual code within the webpage. 
+
 What are attributes?  
+Extra pieces of information that is a modifier of an HTML element. 
+
 What are forms?  
+They provide a way for websites to capture information from users and to process requests
+
 What is a div?  
+Defines a division or a section in an HTML document and is used to group block-elements to format them with CSS
 
 What are selectors?  
+Selectors in CSS refer to element(s) you want to style.
 What are properties?  
+Properties set other visual qualities to your elements. Ususally within a specific selector. 
+
 What are values?  
+Values are the parameters for the specific properties. Examples being hexidecimal color, font size and pixel size.
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+
 What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
 
+What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+1)Type the code within the html file
+2)refrence the CSS file by using a link element.
+3)
 ---
 ### Section 3: Git and Github  
 What is Git?  
+Git is a version Control system that helps keep track of changes in one's code.
+
 What is SCM?  
-What is a VCS?  
+SCM stands for "Supply Chain Management" which is the oversight of information, as they move in a process from supplier to developer and to userer.
+What is a VCS? 
+A version control system is a tool that a developer or group of developers use to keep track of changes with in a project.
+
 Why is Git useful for a developer?  
+Git allows a developer to track their changes in a program and experiment without messing up their working project.
+
 Why is Git useful for a team of developers?  
+Git allows a team of developers to track changes with in their document. It also useful if one wanted to have seperate team members modify certain features in an application. For example they can build seperate branches so they can experiment with the code without messing with the running product. 
+
 How do you create a new Git repository for a project locally?  
+"git init"
```

Comment:
Correct answer, just a note about markdown.  If you put backticks `\`` around text, it gets formatted as code.  Convention is to use this to indicate commands, keywords, variable names, etc.  So:  

How do you create a new Git repository for a project locally?  
`git init`